.github/workflows/code_checks.yml

```diff
@@ -116,15 +117,15 @@ jobs:
           timeout_minutes: 10
           retry_wait_seconds: 3 # Seconds
           max_attempts: 3
-          command: |
-            docker compose -f docker-compose.test.yml run web bash \
-              -c "CI=true RAILS_ENV=test DISABLE_BOOTSNAP=true bundle exec parallel_test -n 24  -e 'bin/rails db:reset'"
+          command: >
+             ${{ env.COMPOSE_BASH_CMD }}
+             "bundle exec rake parallel:setup[24]"
```

Comment:
It's been literally years so I can't remember the reason anymore. But there was a difference between `parallel_test` and `parallel:setup` did you run into the difference when you switched over to this?

Reply:
they're basically the same. it's what they [recommend for rails](https://github.com/grosser/parallel_tests?tab=readme-ov-file#setup-for-rails) and it's a bit cleaner.

Reply:
👀 what Trevor called out is the only change to this command, right? I see these in `docker-compose.test.yml`:
```yaml
      CI: "true"
      RAILS_ENV: test
      DISABLE_BOOTSNAP: "true"
```
So yeah... this should be good!